src/components/news_story/dataService.tsx

```diff
@@ -5,16 +5,15 @@ import { mediaImageDataService } from '@/components/media/dataService'
 import { EntityMetaInfo } from '@/lib/delegators/entityMetaProvider'
```

Comment:
I did not remove this dataService file, but you can think of the `@/data/mappings` and `@/data/queries` files are serving the same function.